### ISSUE TITLE
CEGUI bgfx renderer license

### ISSUE THREAD

**Issue by 3DDean** [charlieamer/SpaceGolfGame#1]
What is the license on the BGFX render? While a bit outdated it has served as a very nice starting point. I need to know what license it is before I can move forward.

**Comment by charlieamer** [comment]
You can use it however you want. I am glad it helped you. Please consider making either a PR, or even better, giving me a link to your code if possible, so that I can copy it to my repo.

**Comment by charlieamer** [comment]
Of course, that is optional :)

**Comment by 3DDean** [comment]
I will happily link it as soon as I fix the issue that I gutted GuiBgfxGeometry to fix, and once i fix the other issue that was introduced by some of the changes I made.

**Comment by 3DDean** [comment]
Did not intend to close the issue or publish the comment quite yet...

**Comment by 3DDean** [comment]
Finally added a public repository. It can be found here. 
https://github.com/3DDean/CEGUI-bgfx-render
Just to be clear, you are OK with commercial use of the library. If so then i would like to put it under the bsd 2-clause simplified license. The same as bgfx.

**Comment by charlieamer** [comment]
Yes, I am totally fine with commercial use of this code. My intention was to provide a code free for anyone. :)

**Comment by charlieamer** [comment]
hey @3DDean how have you been ? :)

I see the library hasn't been updated in 2 years, but I am planning to revisit this project, should I do those as PR on your repo, or you are planning to abandon the project, in which case, I could create repo on my profile ?

Also, can you please add link to your library in this post ? http://cegui.org.uk/forum/viewtopic.php?t=7338 I think people will find your library easier because this post ranks first when you google "cegui bgfx"

**Comment by 3DDean** [comment]
I have no intention of returning to it. Cegui didn't fit my use case. It is probably best if you create a repo on your profile. I do not know how functional it is, as I haven't touched it in at least a year and a half at least. If you have any questions, feel free to ask.

**Comment by charlieamer** [comment]
Out of curiosity what did you end up using ? I am also looking for a nice gui library in games and cegui is the best I could find, but it is still not perfect for me.On Sep 16, 2021 04:25, 3DDean ***@***.***> wrote:
I have no intention of returning to it. Cegui didn't fit my use case. It is probably best if you create a repo on your profile. I do not know how functional it is, as I haven't touched it in at least a year and a half at least. If you have any questions, feel free to ask.

**Comment by 3DDean** [comment]
I switched to imgui, but that is because my use case was more interacting with backend data. Though if I remember correctly imgui is flexible enough to be used for directly for a front end. It is less cumbersome to use then cegui, but again I was primarily focused on interacting with the backend. That project died before reaching a proper end user front end, so I am not quite sure how well it would work, but I am pretty sure it was simpler then cegui. Sorry I can remember more, it has been about 9 months since I have touched that project.
